app/src/main/java/net/skyscanner/backpack/demo/compose/AppSearchModalStory.kt

```diff
@@ -144,7 +145,6 @@ internal fun contentResult() = BpkAppSearchModalResult.Content(
                     subtitle = buildAnnotatedString { append(stringResource(id = R.string.search_modal_item_subtitle)) },
                     icon = BpkIcon.City,
                     onItemSelected = {},
```

Comment:
if you set the `tertiaryLabel` on the first item it'll also appear in the screenshot tests, so we're testing that as well

Reply:
Yeah, But I didn't see the label in first item "current location" on design 🤔️
<img width="274" alt="image" src="https://github.com/Skyscanner/backpack-android/assets/97094570/2370bd38-1eba-4c0b-b4c7-224a185f917c">

Reply:
ah i see - alternatively you could remove one result from `recent searches` and then the first city will appear

Reply:
Ah, updated! Let's see how it turns out. 👀